src/jobs/microk8s_install.yml

```diff
@@ -124,24 +130,88 @@ steps:
         END_VALUES
 
   - run:
-      name: Get chart and setup codacy helm repos
+      name: Setup helm repos
       command: |
         set -x
-        git clone https://github.com/codacy/chart.git
-        cd chart/
-        git checkout << parameters.chart_branch >>
-
         sudo helm repo add codacy-stable    https://charts.codacy.com/stable
         sudo helm repo add codacy-unstable  https://charts.codacy.com/unstable
         sudo helm repo add codacy-incubator https://charts.codacy.com/incubator
         sudo helm repo add codacy-external  https://charts.codacy.com/external
 
-  - deploy:
-      name: Install Codacy to microk8s
+  - when:
+      condition: << parameters.clone_chart_source >>
+      steps:
+        - run:
+            name: Get codacy chart
+            command: |
+              set -x
+              git clone https://github.com/codacy/chart.git
+              cd << parameters.chart_dir_path >>
+              git checkout << parameters.chart_branch >>
+
+  - run:
+      name: Add namespace and docker secret
+      command: |
+        set -x
+        if [[ -z "$(echo $PATH | grep snap)" ]] ; then export PATH=$PATH:/snap/bin ; fi
```

Comment:
I'm installing microk8s from snap, and `/snap/bin` is not in `$PATH`, so I'm adding it to avoid having to write, for instance, the full path to `/snap/bin/microk8s.kubectl` all the time. But I'm cleaning up many steps, and this will only be used in one of the steps, where it is actually needed.